### PR TITLE
format golang code samples and define snippet language formatting

### PR DESCRIPTION
**Pull request by mavimo**
Formatted code snippet using `gofmt` and specify language in codeblock.

You can see changes in `{root_website}/documentation/tutorials/nats-client-dev/`

### CODE REVIEW

File: content/documentation/internals/nats-guide.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ gnatsd -DV -m 8222 -user foo -pass bar
 
 The client must then authenticate to connect to the server. For example:
 
-```
+```go
```
Comment (ColinSullivan1) [inline content/documentation/internals/nats-guide.md]: This did not work for me when I tested this out.

Comment (mavimo) [inline content/documentation/internals/nats-guide.md]: Looking preview on GH seems to work, I'm going to remove it.

**Comment by mavimo** [comment]
@ColinSullivan1 `go` language specifier removed and squashed/renamed commit.

**Comment by ColinSullivan1** [comment]
So the `go` specifier may not have been the issue.  Here's what I see with your PR:

format-golang-page

Do you changes look ok locally running `hugo server -w` and viewing the page at:  http://localhost:1313/documentation/tutorials/nats-client-dev/

**Comment by mavimo** [comment]
@ColinSullivan1 mmmmm, it's not what I can see on my local env.

Without `go` language specifier:
screen shot 2017-06-16 at 08 20 28

With `go` language specifier:
screen shot 2017-06-16 at 08 20 35

This is my `hugo` version
```
marco@nebula ~/Development/mavimo/nats-site patch-1 $ hugo  version
Hugo Static Site Generator v0.22.1 darwin/amd64 BuildDate: 2017-06-16T08:18:06+02:00
```

And this is the command I execute to start display local docs:
```
marco@nebula ~/Development/mavimo/nats-site patch-1 $ hugo serve -w
Started building sites ...
Built site for language en:
0 draft content
0 future content
0 expired content
115 regular pages created
148 other pages created
6 non-page files copied
4 paginator pages created
64 tags created
4 categories created
total in 115 ms
Watching for changes in /Users/marco/Development/mavimo/nats-site/{data,content,layouts,static}
Serving pages from memory
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
```

**Comment by ColinSullivan1** [comment]
@mavimo Interesting.  I wiped everything out, updated hugo, checked out your PR again, and can get it to work.  So, I can verify that it works without the `go` specifier.  

I'll dig further with the go specifier things look very nice.  As is, LGTM.

Your patience is much appreciated!

**Comment by mavimo** [comment]
@ColinSullivan1 np, I'll open a new PR to include go specifier ;)

**Pull request merged**